test/acceptance/features/steps/steps.py

```diff
@@ -301,26 +302,37 @@ def quarkus_app_is_imported_as_knative_service(context, application_name):
     context.application = application
     context.application_type = "knative"
 
+def get_sbr_secret(context):
```

Comment:
this function returns secret name, not secret. Let's rename to reflect that.

Reply:
This is addressed